docs/Logos-Hook-Splitting.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Logos: Hook Splitting"
+layout: docs
+---
+
+By default, the Logos pre-processor will only process one .xm file at build time. However, it is possible to split the Logos hooking code into multiple files.
+
+To do this, the main file has to be renamed to an .xmi file. Then, other .xm files can be included in it using the `#include` or `#import` directives. The Logos pre-processor will add those files to the main file before processing it.
+
+## Groups
+
+Normally, it isn't possible to initialize hooking groups across multiple Logos files, but there is a workaround that can be used to unlock this functionality. This is done by wrapping group initializations inside of static methods that can then be called from other files.
+
+Take a look at the following code. All it does is log a message when the SpringBoard application has finished launching. It is inside of a group called **TweakGroup**, which is initialized in a static function called **InitGroup()**.
+
+```objc
+// Group.xmi
+#import <substrate.h>
+#import "Shared.h"
+#import "Tweak.xm"
```

Comment:
The method being described is an alternative to .xi/.xmi, yet we're using a .xmi file and importing `Tweak.xm`

I say this because using .xi/.xmi simply merges all the files together so the inits for the groups are done after, with the single file. Initializing groups from other files would be possible, no?

Reply:
Whoosh moment. Changes reverted.